geonode_mapstore_client/templatetags/client_version.py

```diff
@@ -0,0 +1,22 @@
+import os
+import logging
+
+from django import template
+
+logger = logging.getLogger(__name__)
+register = template.Library()
+
+
+@register.simple_tag
+def client_version():
+    try:
+        file_path = os.path.join(
+            os.path.join(os.path.dirname(__file__), os.pardir),
```

Comment:
@marthamareal why don't you use settings.STATIC_ROOT?

Reply:
updated